DXRBalance/DeusEx/Classes/AugDisplayWindow.uc

```diff
@@ -254,18 +254,25 @@ function #var(prefix)Teleporter TraceTeleporter(float checkDist, out vector HitL
 
 function string formatMapName(string mapName)
 {
-    local string mapNameOnly,destName;
+    local string mapNameOnly,teleName;
+    local int hashPos;
 
-    if (InStr(mapName,"#")+1 == Len(mapName)) {
+    hashPos = InStr(mapName,"#");
+
+    if (hashPos+1 == Len(mapName)) {
         // # is the last character, leave it out
-        return Left(mapName, Len(mapName)-1);
+        return class'DXRMapInfo'.static.GetTeleporterName(Left(mapName, Len(mapName)-1),"");
     }
-    if (InStr(mapName,"#")==-1){
+    if (hashPos==-1){
         //No # in map name, so it's probably just the map name?
-        return mapName;
+        return class'DXRMapInfo'.static.GetTeleporterName(Left(mapName, Len(mapName)-1),"");
```

Comment:
hmm I don't think this is right for both if statements to have identical substring logic 
```suggestion
        return class'DXRMapInfo'.static.GetTeleporterName(mapName, "");
```